### PR TITLE
add verify email page layout

### LINKED ISSUE #20: Create verify email page layout
Create the layout for Verify Email Page as we discussed earlier.
> Effected file: `/src/pages/authentication/Verify.tsx`.
> Preview url: `/verify-email`

**Only update the markup. No need to implement the logic.**

### PR DESCRIPTION
**Pull request by thatsuman**
Fixes #20 

Didn't add alert notification funtionality, just made the layout.

### CODE REVIEW

File: src/pages/authentication/Verify.tsx [hunk 1]
```diff
@@ -1,5 +1,90 @@
...
+            {/* verify button */}
+            <button
+              type='submit'
+              className='block w-full px-5 py-3 text-sm font-medium text-white bg-blue-800 rounded-lg'
+            >
+              Verify
+            </button>
+          </form>
+
+          {/* alert notification starts*/}
```
Comment (asimbera) [inline src/pages/authentication/Verify.tsx]: No need to add customized notifications. We will add those overlay notifications via some library. No need to waste time here.

File: src/pages/authentication/Verify.tsx [hunk 2]
```diff
@@ -1,5 +1,90 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
```
Comment (asimbera) [inline src/pages/authentication/Verify.tsx]: Please do not keep unused imports.

**Review by asimbera — CHANGES_REQUESTED** [review]

File: src/pages/authentication/Verify.tsx (hunk 1 above)
Comment (thatsuman) [reply src/pages/authentication/Verify.tsx]: Okay I'll remove it by tomorrow.

**Review by asimbera — APPROVED** [review]
Looks good.

**Pull request merged**